pkg/services/ngalert/store/database_mig.go

```diff
@@ -1,12 +1,12 @@
-package ngalert
+package store
 
 import (
 	"fmt"
 
 	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
 )
 
-func addAlertDefinitionMigrations(mg *migrator.Migrator) {
+func AddAlertDefinitionMigrations(mg *migrator.Migrator, defaultIntervalSeconds int64) {
```

Comment:
What's the purpose of the second argument, since it's unused?

Reply:
It is used here:
https://github.com/grafana/grafana/blob/4e62f711cd9ad4815c7d9e8d02772e4f975211e4/pkg/services/ngalert/store/database_mig.go#L21
`addAlertDefinitionMigrations` has moved from the `ngalert` package where this constant is defined.

Reply:
@papagian Ah, I expected to see the parameter's usage in the diff. Thanks!